src/Auth/Eloquent/UserRepository.php

```diff
@@ -34,11 +35,13 @@ public function all(): UserCollection
 
     public function find($id): ?UserContract
     {
-        if ($model = $this->model('find', $id)) {
-            return $this->makeUser($model);
-        }
+        return Blink::once("eloquent-user-find-{$id}", function () use ($id) {
+            if ($model = $this->model('find', $id)) {
+                return $this->makeUser($model);
+            }
 
-        return null;
+            return null;
+        });
     }
 
     public function findByEmail(string $email): ?UserContract
```

Comment:
Not sure whether or not we should maybe be doing the same Blink thing here? 

I know this method won't be called for the `last_updated` user but maybe doing it here too would be good to keep it consistent?

Reply:
Sure, why not? 😄